mongodbforms/fields.py

```diff
@@ -80,16 +80,22 @@ class ReferenceField(forms.ChoiceField):
     """
     widget = LazySelect
 
-    def __init__(self, queryset, empty_label="---------", *args, **kwargs):
+    def __init__(
+            self,
+            queryset,
+            empty_label="---------",
+            label_formatter=smart_unicode,
+            **kwargs):
         # skip choice initialization in forms.ChoiceField.__init__
-        forms.Field.__init__(self, *args, **kwargs)
+        forms.Field.__init__(self, **kwargs)
```

Comment:
This means that `forms.Field` isn't passed `queryset`, `empty_label` or `label_formatter` anymore... Is that correct?

Reply:
Django only specifies `**kwargs`, so I guess it's fine.

Reply:
Sorry, I didn't see your question. Thanks for merging. The arguments weren't ever passed into the `__init__` since they are specified explicitly and `*args` would be grabbing anything else. Looking at django it only takes `**kwargs` so I figured it wouldn't be a breaking change.